docs/package.json

```
@@ -1,33 +1,37 @@
 {
   "name": "remeda-docs",
   "version": "1.0.0",
-  "main": "index.js",
   "repository": "git@github.com:remeda/docs.git",
   "author": "Łukasz Sentkiewicz",
   "license": "MIT",
   "scripts": {
-    "start": "node index",
-    "watch": "parcel serve index.html",
-    "build": "cross-env NODE_ENV=production parcel build index.html --public-url ./  --out-file index.html && cp _redirects dist/_redirects && cp dist/index.html dist/404.html"
+    "watch": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```

1. Put `reportUnusedDisableDirectives` in the config itself
2. We don't need to mention the extensions here, they should be part of the config too, and generally this isn't required
3. We don't need the 0 warnings here, it's something we can turn on in the CI to specifically prevent these from falling through, for the user it doesn't matter if something is a warning or an error
```suggestion
    "lint": "eslint",
```

---

I'm trying to remove the eslint module from the docs folder. So I'm thinking of writing the actual lint code in root's package.json. From the docs folder, I'm thinking of replacing it with the following scirpts

docs/packages.json
```json
"lint": "cd ../ && yarn docs:lint",
```

./packages.json
```json
"docs:lint": "eslint docs/ -c docs/.eslint.cjs"
```

---

https://github.com/remeda/remeda/pull/359/commits/e0717c77f2e8290edbf8646b2cbe14436967e9d8

eslint that can be organized in the root folder is organized in the root folder